voicevox_engine/tts_pipeline/njd_feature_processor.py

```diff
@@ -0,0 +1,74 @@
+"""NJD Featureの処理"""
+
+import re
+from dataclasses import asdict, dataclass
+
+import pyopenjtalk
+
+
+@dataclass
+class NJDFeature:
```

Comment:
OjtPhonemeと同じ感じで、こうしたいかも
```suggestion
class NjdFeature:
```

Reply:
https://github.com/VOICEVOX/voicevox_engine/pull/1562/commits/4981c01062a2cc95de150b185bd2cddc563c3e0c にて変更済です。